app/api/templates/specs/fixtures/fixtures.js

```diff
@@ -43,47 +62,62 @@ export default {
       commonProperties: [{ name: 'title', label: 'Title', type: 'text' }],
     },
     {
-      _id: db.id(),
+      _id: thesaurusTemplateId,
       name: 'thesauri template',
       properties: [
         {
+          _id: db.id(),
           type: propertyTypes.select,
           content: thesauriId1.toString(),
           label: 'select',
           name: 'select',
         },
-        { type: propertyTypes.relationship, content: templateToBeDeleted, label: 'select2' },
+        {
+          _id: db.id(),
+          type: propertyTypes.relationship,
+          content: templateToBeDeleted,
+          label: 'relationshipToBeDeleted',
+          name: 'relationshipToBeDeleted',
+        },
       ],
-      commonProperties: [{ name: 'title', label: 'Title' }],
+      commonProperties: [{ name: 'title', label: 'Title', type: 'text' }],
     },
     {
-      _id: db.id(),
+      _id: thesaurusTemplate2Id,
       name: 'thesauri template 2',
       properties: [
         {
+          _id: db.id(),
           type: propertyTypes.select,
           content: thesauriId1.toString(),
           label: 'select2',
           name: 'select2',
         },
-        { type: propertyTypes.select, content: templateToBeDeleted, label: 'selectToBeDeleted' },
+        {
+          _id: db.id(),
+          type: propertyTypes.select,
+          content: templateToBeDeleted,
+          label: 'selectToBeDeleted',
+          name: 'selectToBeDeleted',
+        },
       ],
-      commonProperties: [{ name: 'title', label: 'Title' }],
+      commonProperties: [{ name: 'title', label: 'Title', type: 'text' }],
     },
     {
```

Comment:
Is this case even possible? How can it happen that you have a select property and it's content is a template and not a thesauri?

Reply:
That is a good question. Since these were existing tests, I've just updated them assuming there were there for a reason. Let's also ask this from the team.

Reply:
Tests updated according to the discussion with @daneryl .